### PR TITLE
fix(#293): updated parsing of UDP information from URL

### PR DESCRIPTION
**Pull request by JanssenBrm**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -403,6 +403,10 @@ class Utils extends VueUtils {
 		};
 		return resolver(schema);
 	}
+	static extractUDPParams(process) {
+		let [id, ...namespace] = process.split('@');
+		return [id, namespace.join('@')]
```
Comment (m-mohr) [inline src/utils.js:408]:
```suggestion
		return [id, namespace.join('@')];
```

Interesting how "stupid" such a code looks in JS while in other languages it's just a matter of providing a limit to split.

**Review by m-mohr — CHANGES_REQUESTED** [review]
Thanks. This is looking good, but it also needs to be fixed here:
https://github.com/Open-EO/openeo-web-editor/blob/98524783513d7896448f2974b02a3a3b9b3c63d9/src/store/editor.js#L49

File: src/utils.js (hunk 1 above)
Comment (JanssenBrm) [reply src/utils.js:408]: These little differences between programming languages are always tricky. In this case, the code indeed looks rather "stupid" to just make a simple split work.

**Review by m-mohr — APPROVED** [review]

**Comment by m-mohr** [comment]
Thanks

**Pull request merged**